src/region/RegionCreator.tsx

```diff
@@ -183,10 +198,13 @@ export default class RegionCreator extends React.PureComponent<Props, State> {
 
     render(): JSX.Element {
         const { canDraw, className } = this.props;
+        const { isMouseOver } = this.state;
```

Comment:
This component may end up changing quite a bit soon to help with performance: https://github.com/box/box-annotations/pull/490